Sources/OLEKit/FAT.swift

```diff
@@ -36,16 +36,16 @@ enum SectorID: UInt32 {
  FAT sectors, right after the header which is 76 bytes long.
  (always 109, whatever the sector size: 512 bytes = 76+4*109)
  Additional sectors are described by DIFAT blocks */
-private let maxFATSectorsCount: UInt32 = 109
+let maxFATSectorsCount: UInt32 = 109
```

Comment:
Can this be kept `private`? Or if not, maybe `fileprivate` would work?

```suggestion
private let maxFATSectorsCount: UInt32 = 109
```

Reply:
Unfortunately, it could be fileprivate because extension Reader can't have additional property. I changed it to commit f8de875.